tests/rgen/Microsoft.Macios.Generator.Tests/Classes/Data/ExpectedAVAudioPcmBufferDefaultCtr.cs

```diff
@@ -0,0 +1,46 @@
+// <auto-generated />
+
+#nullable enable
+
+using Foundation;
+using ObjCBindings;
+using ObjCRuntime;
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using System.Threading.Tasks;
+
+namespace TestNamespace;
+
+[Register ("AVAudioPCMBuffer", true)]
+public partial class AVAudioPcmBuffer
+{
+	[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
+	static readonly NativeHandle class_ptr = Class.GetHandle ("AVAudioPCMBuffer");
+
+	public override NativeHandle ClassHandle => class_ptr;
+
```

Comment:
You're not generating the XML docs for the constructors, but I guess that can come in a different PR.

Reply:
That's the plan. Specially because I don't know yet an elegante way to do it :)